cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -323,7 +337,7 @@ predicate simpleLocalFlowStep(Node nodeFrom, Node nodeTo) {
   simpleInstructionLocalFlowStep(nodeFrom.asInstruction(), nodeTo.asInstruction())
 }
 
-private predicate simpleInstructionLocalFlowStep(Instruction iFrom, Instruction iTo) {
+predicate simpleInstructionLocalFlowStep(Instruction iFrom, Instruction iTo) {
```

Comment:
Why no longer private?

Reply:
Oops! That was a leftover from my debugging that slipped into a commit.